app/models/analyticEvent.server.ts

```diff
@@ -1,56 +1,187 @@
-import { AnalyticEvent } from "@prisma/client";
+import { AnalyticEvent, AnalyticEventByDay } from "@prisma/client";
 import { prisma } from "~/db.server";
 
 export async function addAnalyticEvent(
-    ip: string,
-    event: string,
-    data: string
-  ): Promise<void> {
-    await prisma.analyticEvent.create({
-      data: {
-        ip: ip,
-        event: event,
-        data: data,
-      },
-    });
-  }
+  ip: string,
+  event: string,
+  data: string,
+  ts: number
```

Comment:
make this optional, with default the current time

Reply:
Resolved.